packages/app-project/pages/404.js

```diff
@@ -1,115 +1,131 @@
-          height="80vh"
-          justify="center"
-        />
-        <PageContent
-          alignContent="center"
-          justify="center"
-          textAlign="center"
-        >
-          <Paragraph textAlign="center" style={{ margin: 0 }}>
-            <Image
-              id="404-logo"
-              fit="contain"
-              height="44"
-              width="132"
-              alt="404"
-              src={`/projects/assets/logoWhite404.png`}
-            />
-          </Paragraph>
-          <Heading
-            level="1"
-            textAlign="center"
-            margin={{ top: '20px', bottom: '20px' }}
-            style={{ maxWidth: '100%' }}
-          >{t('404.heading')}</Heading>
-          <Paragraph textAlign="center">
-            <Text style={{ fontStyle: 'italic', display: 'block', marginBottom: '30px' }}>{t('404.message')}</Text>
-            <Text style={{ display: 'block', marginTop: '30px', marginBottom: '15px' }}>• <Anchor
-                id="404-to-project-home"
-                color="white"
-                weight="bold"
-                href={`/`}
-                label={t('404.returnHome')}
-                style={{ textDecoration: 'underline' }}
-              />
-            </Text>
-            <Text style={{ display: 'block', marginTop: '15px' }}>•&nbsp;
-              <Anchor
-                id="404-to-project-home"
-                color="white"
-                weight="bold"
-                href={`/projects`}
-                label={t('404.findNewProject')}
-                style={{ textDecoration: 'underline' }}
-              />
-            </Text>
+      <ContainerBox width='100%' height='80vh' justify='center'>
+        <Overlay background={`url("${backgroundURL}")`} fill />
+        <PageContent justify='center' align='center'>
+          <Image
+            id='404-logo'
+            width={logoSize(size)}
+            alt='404 logo'
+            src='/projects/assets/logoWhite404.png'
+            margin={{ bottom: 'xxsmall' }}
+          />
+          <Heading level='1' textAlign='center' fill color='white' size={size}>
+            {t('404.heading')}
+          </Heading>
+          <Paragraph textAlign='center' color='white'>
+            <em>{t('404.message')}</em>
           </Paragraph>
+          <Box as='ul' fill align='center' margin='0' pad='0' gap='10px'>
+            <StyledAnchor
+              forwardedAs='li'
+              color='white'
+              href='https://www.zooniverse.org'
+              label={t('404.returnHome')}
+              size='1rem'
+            />
+            <StyledAnchor
+              forwardedAs='li'
+              color='white'
+              href='https://www.zooniverse.org/projects'
+              label={t('404.findNewProject')}
+              size='1rem'
+            />
+          </Box>
         </PageContent>
       </ContainerBox>
       <ZooFooter locale={locale} />
     </ThemeContext.Extend>
   )
 }
+
+const customTheme = {
```

Comment:
I think purely aesthetic, but I typically see constants used in the functional component declared before the functional component.